test-harness/test-harness-spec.yaml

```diff
@@ -0,0 +1,303 @@
+openapi: 3.0.1
+
+info:
+  title:
+     Fn::Sub: "${AWS::StackName} - F2F Credential Issuer Private API"
+  version: "1.0"
+  description: F2F Test automation harness API specification
+
+x-amazon-apigateway-globals:
+  servers:
+    - url: https://www.example.com
+
+tags:
+  - name: Testing - API to enable automation
+    description: Endpoint implemented to enable testing
+
+# x-amazon-apigateway-policy:
```

Comment:
Is this to be enhanced separately?

Reply:
yes, see https://gds.slack.com/archives/C0440GZHUUX/p1691581377069459 Gabby is looking into it

Reply:
yes, see https://gds.slack.com/archives/C0440GZHUUX/p1691581377069459 Gabby is looking into it